### PR TITLE
[Math] Add a solution to Permutation Sequence

### PR DESCRIPTION
**Pull request by WAMaker**

### CODE REVIEW

File: Math/PermutationSequence.swift [hunk 1]
```diff
@@ -0,0 +1,39 @@
+/**
+ * Question Link: https://leetcode.com/problems/permutation-sequence/
+ * Primary idea: Iterate and change the array from last to the first
+ *
+ * Time Complexity: O(n^2), Space Complexity: O(1)
+ */
+
+class PermutationSequence {
+	func getPermutation(_ n: Int, _ k: Int) -> String {
```
Comment (soapyigu) [inline Math/PermutationSequence.swift]: Your indention looks a little weird. Seems the indent width is 7 spaces? Could you change it to 2 or 4 and reformat your code? Thanks!

Comment (WAMaker) [reply Math/PermutationSequence.swift]: Wired problem. It looks good to me in VSCode. I fixed this and will check my IDE setting later.

**Comment by WAMaker** [comment]
Already upgraded `progress`. Please check.

**Comment by soapyigu** [comment]
Hi Sorry seems there is a conflict in README file. I guess you have to update the question number we have resolved in `progress` session. Thanks.

**Comment by WAMaker** [comment]
Already Updated the `progress` to 276.

**Pull request merged**